pygmt/modules.py

```diff
@@ -55,7 +55,7 @@ def grdinfo(grid, **kwargs):
 
 @fmt_docstring
 @use_alias(C="per_column", I="spacing", T="nearest_multiple")
-def info(fname, **kwargs):
+def info(table, **kwargs):
```

Comment:
There's actually an API change here (argument name change from `fname` to `table`), and maybe we should put this into v0.2.0?

Reply:
It looks a little weird that the new `info()` function support pandas.DataFrame, but not 2D numpy arrays, and the return value `-R-180/180/-65.7/86.8\n` looks useless for Python.

Reply:
> It looks a little weird that the new `info()` function support pandas.DataFrame, but not 2D numpy arrays

Yes, I could make it support 2D numpy arrays too, lemme work on that.

> and the return value `-R-180/180/-65.7/86.8\n` looks useless for Python.

That's dealt with in a parallel PR at #575 (which is branched off this PR).

Reply:
>> It looks a little weird that the new info() function support pandas.DataFrame, but not 2D numpy arrays

>Yes, I could make it support 2D numpy arrays too, lemme work on that.

Just on this, does gmt `info` work on single column data as well? i.e. 1D arrays?

Reply:
Yes. For a one-column data,
```
1
2
3
4
5
```
```
gmt info test.txt -I1  
-R1/5/0/0
```

Reply:
Ok, will support this too then.